baqend/getTestOverviewReport.ts

```diff
@@ -67,22 +68,37 @@ async function fetchAllTestOverviews(
  * @returns {SortedDomain[]} - An array of sorted domains with their counts and statuses.
  */
 function calculateDomainStatus(testOverviews: TestOverview[]): SortedDomain[] {
-  const domainStatusCount: Record<string, { count: number, statuses: Record<string, number> }> = {};
+  const urlStatusCount: Record<string, { count: number, statuses: Record<string, number>, lastTested: Date }> = {};
 
   testOverviews.forEach(entry => {
-    const origin = extractOrigin(entry.url);
-    const processedOrigin = processOrigin(origin);
-    if (processedOrigin && !EXCLUDED_DOMAINS.includes(processedOrigin)) {
-      if (!domainStatusCount[origin]) {
-        domainStatusCount[origin] = { count: 0, statuses: {} };
+    const fullUrl = entry.url;
+    if (!EXCLUDED_DOMAINS.some(domain => fullUrl.includes(domain))) {
```

Comment:
Wollen wir das hier in ein early return umbauen? Dann reduzieren wir die Verschachtelung etwas.